### PR TITLE
create light, dark variants of chec-loading

### PR DESCRIPTION
**Pull request by john-raymon**
ezgif-6-ec73289b66a2

### CODE REVIEW

**Comment by dvnkshl** [comment]
@john-raymon are you able to reverse the animation via css/code? It looks like its loading backwards (going left to right vs right to left lol)

**Comment by john-raymon** [comment]
@dvnkshl flipped
ezgif-6-8914d881ece0 (1)

**Comment by dvnkshl** [comment]
@blazepollard might need to update the ui lib? Just for consistency

**Comment by robbieaverill** [comment]
I thought that was because we're in the northern hemisphere

File: src/components/ChecLoading.vue [hunk 1]
```diff
@@ -43,50 +37,104 @@ export default {
...
+    variant: {
+      type: String,
+      default: 'default',
+      validator: function validator(value) {
+        return ['dark', 'light'].includes(value);
+      },
+    },
+    withoutBackground: {
+      type: Boolean,
+    },
```
Comment (ScopeyNZ) [inline src/components/ChecLoading.vue:49]:
```suggestion
    withoutBackground: Boolean,
```

**Pull request merged**